build/bin/copy-role-docs.sh

```diff
@@ -71,3 +71,4 @@ copyDoc suite_mustgather_download
 copyDoc suite_upgrade
 copyDoc suite_verify
 copyDoc uds
+copyDoc cis
```

Comment:
add this in alphabetical order like the rest please.  Also add this to appropriate section in mkdocs.yml